README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains repo contains Chiemeziem's portfolio and SPS projects.
```

Comment:
It looks like "repo contains" ended up twice here.

Reply:
I have fixed the redundancy in the README file.